pkg/tsdb/influxdb/response_parser.go

```diff
@@ -114,8 +114,10 @@ func transformRows(rows []Row, query Query) data.Frames {
 
 				for _, valuePair := range row.Values {
 					timestamp, timestampErr := parseTimestamp(valuePair[0])
+					value := valuePair[1]
+
 					// we only add this row if the timestamp is valid
-					if timestampErr == nil {
+					if timestampErr == nil && value != nil {
```

Comment:
So this is a huge breaking change, and probably won't fly, but this does indeed solve the problems in merging dataframes in incremental querying

Reply:
Also this makes the responses a good bit smaller